src/js/themes/base.js

```diff
@@ -776,6 +776,7 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         units: {
           color: 'text-xweak',
           margin: { left: 'xsmall' },
+          alignSelf: 'end',
```

Comment:
Just confirming I follow we this is here -- does this cover what was previously being handled by `align="baseline"`?

Reply:
the `align="baseline"` was not impacting the styling of the units but i believe it was trying to achieve the results accomplished through aligning the units at the end of the container so when wrapped, the units are end aligned